test/lint/lint_ignore_dirs.py

```
@@ -0,0 +1,8 @@
+SHARED_EXCLUDED_SUBTREES = ["src/leveldb/",
+                 "src/crc32c/",
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)